### DISCUSSION TITLE
Quickly losing dynamic memory with strings

### DISCUSSION THREAD

**Discussion by jehoshua-dev** [VasilKalchev/LiquidMenu#73] [Ideas]
LiquidMenu is very convenient, although using strings with it quickly eats up RAM on my Arduino Uno. Would it be possible to use LiquidMenu with strings held in program memory, or is there any other solution to avoid the high memory cost of LiquidMenu's strings on dynamic memory?

Edit: Is set_asProgmem() an option?

**Comment by VasilKalchev** [comment]
Yes, `set_asProgmem(uint8_t)` is the solution, see #2.
The feature was in beta, now is available in [release v1.1.6](https://github.com/VasilKalchev/LiquidMenu/releases/tag/v1.1.6).
